webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.tsx

```diff
@@ -31,7 +31,16 @@ import LDAPTextSetting from './ldap_text_setting';
 import {ldapWizardAdminDefinition} from '../admin_definition';
 import {getConfigFromState, isSetByEnv, SchemaAdminSettings} from '../schema_admin_settings';
 import SchemaText from '../schema_text';
-import type {AdminDefinitionSetting, AdminDefinitionSettingButton, AdminDefinitionSettingFileUpload, AdminDefinitionSubSectionSchema, ConsoleAccess} from '../types';
+import type {AdminDefinitionConfigSchemaSection, AdminDefinitionSetting, AdminDefinitionSettingButton, AdminDefinitionSettingFileUpload, AdminDefinitionSubSectionSchema, ConsoleAccess} from '../types';
+import './ldap_wizard.scss';
+
+export type LDAPAdminDefinitionConfigSchemaSettings = AdminDefinitionSubSectionSchema & {
+    sections?: LDAPAdminDefinitionConfigSchemaSection[];
+}
+
+export type LDAPAdminDefinitionConfigSchemaSection = AdminDefinitionConfigSchemaSection & {
+    sectionTitle?: string;
```

Comment:
Trying this out, having some sections with a `title` and `subtitle` while others have a `title` and a `sectionTitle` seems like a weird inconsistency to me. The two sections where we're using this feel kind of mismatched because the headings don't match the button in the sidebar.

<img width="817" alt="Screenshot 2025-05-23 at 5 08 29 PM" src="https://github.com/user-attachments/assets/3fe9570d-d3d8-49d8-bb2d-03beb6ba0884" />

Perhaps we should make those consistent and move the other text into a subtitle?